assets/chemistry-grammar.ne

```diff
@@ -51,7 +51,7 @@ const lexer = moo.compile({
     RSquare: "]",
 
     // Electrons
-    Electron: /(?:e|\\electron)\^{-}/,
+    Electron: /(?:e|\\electron)(?:\^{-}|-)?/,
```

Comment:
Good point, I haven't run those tests for a while 😓. But yes all the changes are intentional the tests have just not been updated, the original behaviour was based on how I thought it should work rather than the eccentricities of how the whole system works.